### DISCUSSION TITLE
ADC Temperature Alexa

### DISCUSSION THREAD

**Discussion by fpgiovanini** [arendst/Tasmota#22207] [Support request]
Hi support Team.
How Can I have alexa answering the temperature read from ADC?
I enabled the adc pin.Console shows the temperature but how can I create a friendly name in configure others to enable alexa answers?
tks, best regards

**Comment by sfromis** [comment]
I have no answer (not using Alexa), but do notice that the [docs about the Alexa integration](https://tasmota.github.io/docs/Alexa/) via Hue has a note:
> "Hue Bridge emulation does not support sensors"

**Reply by fpgiovanini** [reply]
thanks!!!

**Comment by s-hadinger** [comment]
Alexa/Hue emulation supports only plugs and lights.

If your device has an ESP32, you can enable Matter which is supported on most Alexa gateways. Matter on Alexa is supposed to support temperature, although I have not tested

**Reply by fpgiovanini** [reply]
thanks!!
